src/getCoinCombination.test.js

```diff
@@ -7,4 +7,37 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+
+  it('should be non-negative input integer', () => {
+    expect(() => getCoinCombination(-1)).toThrow();
+  });
+
+  it(`should return array with length '4'`, () => {
+    expect(Array.isArray(getCoinCombination(1))).toBe(true);
+    expect(getCoinCombination(1)).toHaveLength(4);
+  });
+
+  it('should return [0, 0, 0, 0] for 0 cents', () => {
+    expect(getCoinCombination(0)).toEqual([0, 0, 0, 0]);
+  });
+
+  it(`should return '[1, 0, 0, 0]' for 1 cents`, () => {
+    expect(getCoinCombination(1)).toEqual([1, 0, 0, 0]);
+  });
+
+  it(`should return '[1, 1, 0, 0]' for 6 cents`, () => {
+    expect(getCoinCombination(6)).toEqual([1, 1, 0, 0]);
+  });
+
+  it(`should return '[1, 1, 1, 0]' for 16 cents`, () => {
+    expect(getCoinCombination(16)).toEqual([1, 1, 1, 0]);
```

Comment:
The test description 'should return '[1, 1, 1, 0]' for 16 cents' is incorrect based on typical US coin denominations. The expected result should likely be '[1, 1, 0, 1]' for 16 cents, assuming the denominations are pennies, nickels, dimes, and quarters.

Reply:
?? 1 dime = 10 cents > 16 - 10 = 6
?? 5 nikel = 5 cents > 6 - 5 = 1
?? 1 cent = 1 cent
 result [1cent, 1nikel, 1dime, 0quarter]